src/AdminService/Startup.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace AdminService
+{
+  public class Startup
+  {
+    public Startup(IConfiguration configuration)
+    {
+      Configuration = configuration;

```

Comment:
add BaseServiceInfoConfig
RabbitMQConfig, and Configure mass transit
add token middleware
add connection to db

Reply:
done